index.html

```diff
@@ -2,23 +2,35 @@
 <html>
 
 <head>
-    <title>Pizza Party</title>
-    <link href="style.css" type="text/css" rel="stylesheet">
+
+    <body>
+
+        <div class="header">
+            <div class="logo">
+                <a href="index.html" class="logo">
+                    <img src="img/logo.jpg" alt="logo">
+                </a>
+            </div>
+            <h1>Pizza Party
+
+                <div class="topnav">
+                    <a href="./ourstory.html ">Our Story</a>
+                    <a href="./Menu.html ">Menu</a>
+                    <a href="./Reservations.html ">Reservations</a>
+                    <a href="./contactus.html ">Connect with Us!</a>
+                </div>
+            </h1>
+        </div>
```

Comment:
These lines should not be _inside_ of `<head>` tag

Reply:
this is fixed